lab1/main_test.js

```diff
@@ -3,21 +3,35 @@ const assert = require('assert');
 const { MyClass, Student } = require('./main');
 
 test("Test MyClass's addStudent", () => {
-    // TODO
-    throw new Error("Test not implemented");
+    const myClass = new MyClass();
+    assert.equal(myClass.addStudent(69), -1);
+    const student = new Student();
+    const name = "John";
+    student.setName(name);
+    myClass.addStudent(student);
 });
 
 test("Test MyClass's getStudentById", () => {
-    // TODO
-    throw new Error("Test not implemented");
+    const myClass = new MyClass();
+    const student = new Student();
+    const name = "John";
+    student.setName(name);
+    const id = myClass.addStudent(student);
+    assert.strictEqual(myClass.getStudentById(-1), null);
+    assert.strictEqual(myClass.getStudentById(50), null);
+    assert.strictEqual(myClass.getStudentById(id), student);
 });
 
 test("Test Student's setName", () => {
-    // TODO
-    throw new Error("Test not implemented");
+    const student = new Student();
+    student.setName(69);
+    student.setName("John");
```

Comment:
You should add assertions after you invoke the unit you want to test to assert that they behave as your expectation.